lib/simple-seeder.js

```diff
@@ -156,12 +160,15 @@ module.exports = class SimpleSeeder {
         }
       }
 
-      for await (const e of snap.createReadStream()) {
+      for await (const e of snap.createReadStream(contentEncoding)) {
         if (e.value.type === 'list') continue // List of lists is not supported
 
         await this.put(e.key, { ...e.value, external: false })
       }
 
+      const allowedPeers = await list.metadata.get(ALLOWED_PEERS)
```

Comment:
Think you have to use `snap`, similar to what you did with the `createReadStream`

```js
const entry = await snap.get(ALLOWED_PEERS, metadataEncoding)
info.userData.allowedPeers = entry ? entry.value : null
```

It's possible to re-create the SeedBee object to avoid encoding ourselves but this is way simpler